exercises/triangle/src/test/java/TriangleTest.java

```diff
@@ -51,75 +68,77 @@ public void isoscelesTrianglesHaveTwoFirstSidesEqual() throws TriangleException
-        Triangle triangle = new Triangle(10, 10, 2);
+    public void isoscelesTrianglesCanHaveAllSidesEqual() throws TriangleException {
+        Triangle triangle = new Triangle(4, 4, 4);
 
         assertTrue(triangle.isIsosceles());
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void scaleneTrianglesHaveNoEqualSides() throws TriangleException {
-        Triangle triangle = new Triangle(3, 4, 5);
+    public void isoscelesTrianglesMustHaveAtLeastTwoEqualSides() throws TriangleException {
+        Triangle triangle = new Triangle(2, 3, 4);
 
-        assertTrue(triangle.isScalene());
+        assertFalse(triangle.isIsosceles());
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void scaleneTrianglesHaveNoEqualSidesAtLargerScaleEither() throws TriangleException {
-        Triangle triangle = new Triangle(10, 11, 12);
+    public void verySmallTrianglesCanBeIsosceles() throws TriangleException {
+        Triangle triangle = new Triangle(0.5, 0.4, 0.5);
 
-        assertTrue(triangle.isScalene());
+        assertTrue(triangle.isIsosceles());
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void scaleneTrianglesHaveNoEqualSidesInDescendingOrderEither() throws TriangleException {
-        Triangle triangle = new Triangle(5, 4, 2);
+    public void scaleneTrianglesHaveNoEqualSides() throws TriangleException {
+        Triangle triangle = new Triangle(5, 4, 6);
 
         assertTrue(triangle.isScalene());
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void verySmallTrianglesAreLegal() throws TriangleException {
-        Triangle triangle = new Triangle(0.4, 0.6, 0.3);
+    public void trianglesWithAllSidesEqualAreNotScalene() throws TriangleException {
+        Triangle triangle = new Triangle(4, 4, 4);
 
-        assertTrue(triangle.isScalene());
+        assertFalse(triangle.isScalene());
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void trianglesWithNoSizeAreIllegal() throws TriangleException {
-        expectedException.expect(TriangleException.class);
-        new Triangle(0, 0, 0);
+    public void trianglesWithOneUnequalSideAreNotScalene() throws TriangleException {
+        Triangle triangle = new Triangle(4, 4, 3);
+
+        assertFalse(triangle.isScalene());
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void trianglesWithNegativeSidesAreIllegal() throws TriangleException {
-        expectedException.expect(TriangleException.class);
-        new Triangle(3, 4, -5);
+    public void verySmallTrianglesCanBeScalene() throws TriangleException {
+        Triangle triangle = new Triangle(0.5, 0.4, 0.6);
+
+        assertTrue(triangle.isScalene());
     }
 
     @Ignore("Remove to run test")
     @Test
-    public void trianglesViolatingTriangleInequalityAreIllegal() throws TriangleException {
+    public void trianglesWithNoSizeAreIllegal() throws TriangleException {
```

Comment:
Please remove `throws TriangleException` from this test method signature.

Reply:
Please relocate this test so that it matches the order specified in the canonical tests.